test/test_permissions_api.js

```diff
@@ -0,0 +1,52 @@
+import should from 'should';
+import supertest from 'supertest';
+
+import app from '../src/server';
+import db from '../src/models/db';
+import config from '../src/config';
+
+import { loginAsAdmin } from './common';
+
+const endpointPrefix = '/' + config.get('version');
+const server = supertest.agent(app);
```

Comment:
nit: using an `agent` is only useful if you want to keep cookies between calls, which we don't need here. Moreover defining it here actually keeps a state between tests which is very wrong™️.

Just use `const server = supertest(app);` like I did in `test_auth_api` :)